docs/core/plugins/push_notification.md

```diff
@@ -136,15 +141,15 @@ Push notification sent in JSON format with the following fields:
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| `event` | string | [[rfc,5423]] event type (currently only "MessageNew") |
-| `folder` | string | Mailbox name |
-| `from` | string | [[rfc,2822]] address of the message sender (MIME-encoded), if applicable |
-| `imap-uid` | integer | UID of the message, if applicable |
-| `imap-uidvalidity` | integer | [[rfc,3501]] UIDVALIDITY value of the mailbox |
-| `snippet` | string | Snippet of the message body (UTF-8), if applicable |
-| `subject` | string | Subject of the message (MIME-encoded), if applicable |
-| `unseen` | integer | [[rfc,3501]] UNSEEN value of the mailbox |
-| `user` | string | User identifier |
+| `event` | [[link,settings_types_string]] | [[rfc,5423]] event type (currently only "MessageNew") |
+| `folder` | [[link,settings_types_string]] | Mailbox name |
+| `from` | [[link,settings_types_string]] | [[rfc,2822]] address of the message sender (MIME-encoded), if applicable |
+| `imap-uid` | [[link,settings_types_uint]] | UID of the message, if applicable |
```

Comment:
OK. Shouldn't they then be "number" and not "integer" as JSON does not contain the "integer" type.

Reply:
I guess it depends on whether it's talking about JSON types specifically or the kind of data there is. Which one is more useful? I guess it could be made explicit with "number (0..4294967295)" but seems a bit unnecessary.

Reply:
I added a note saying that the numbers are of a specific, expected value range. This should be both correct, overly pedantic and not too intrusive.